src/openapi.yaml

```diff
@@ -3451,6 +3451,207 @@ paths:
+                  $ref: '#/components/schemas/InformatieObjectType'
+          description: ''
+          headers:
+            X-Session-Expires-In:
+              $ref: '#/components/headers/X-Session-Expires-In'
+            X-CSRFToken:
+              $ref: '#/components/headers/X-CSRFToken'
+            X-Is-Form-Designer:
+              $ref: '#/components/headers/X-Is-Form-Designer'
+            Content-Language:
+              $ref: '#/components/headers/Content-Language'
+  /api/v2/objects-api/object-types:
+    get:
+      operationId: objects_api_object_types_list
+      description: |-
+        List the available Objecttypes.
+
+        Note that the response data is essentially proxied from the configured Objecttypes API.
+      parameters:
+      - in: query
+        name: objects_api_group
+        schema:
+          type: string
+        description: Which Objects API group to use.
+      tags:
+      - registration
+      security:
+      - cookieAuth: []
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Objecttype'
+          description: ''
+          headers:
+            X-Session-Expires-In:
+              $ref: '#/components/headers/X-Session-Expires-In'
+            X-CSRFToken:
+              $ref: '#/components/headers/X-CSRFToken'
+            X-Is-Form-Designer:
+              $ref: '#/components/headers/X-Is-Form-Designer'
+            Content-Language:
+              $ref: '#/components/headers/Content-Language'
+  /api/v2/objects-api/object-types/{objecttype_uuid}/versions:
+    get:
+      operationId: objects_api_object_types_versions_list
+      description: |-
+        List the available versions for an Objecttype.
+
+        Note that the response data is essentially proxied from the configured Objecttypes API.
+      parameters:
+      - in: query
+        name: objects_api_group
+        schema:
+          type: string
+        description: Which Objects API group to use.
+      - in: path
+        name: objecttype_uuid
+        schema:
+          type: string
+          format: uuid
+        required: true
+      tags:
+      - registration
+      security:
+      - cookieAuth: []
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ObjecttypeVersion'
+          description: ''
+          headers:
+            X-Session-Expires-In:
+              $ref: '#/components/headers/X-Session-Expires-In'
+            X-CSRFToken:
+              $ref: '#/components/headers/X-CSRFToken'
+            X-Is-Form-Designer:
+              $ref: '#/components/headers/X-Is-Form-Designer'
+            Content-Language:
+              $ref: '#/components/headers/Content-Language'
+  /api/v2/objects-api/target-paths:
```

Comment:
I think this endpoint should remain specific to the objects API registration plugin, because it requires the particular input prameters.